### PR TITLE
Added new page with 2 different possibilities

### PR DESCRIPTION
**Pull request by tykiblood**

### CODE REVIEW

**Comment by tykiblood** [comment]
Please review this, @casey-collab

File: espanol/canudos.md [hunk 1]
```diff
@@ -154,3 +154,6 @@ Y llego [Daynerys Targaryen](thrones/daynerys.md)
 
 Recordó algunos crímenes que pesaban sobre él y de la nada perdió el conocimiento(desmayo/desmayo.md)
 
+Revelo que venia del futuro y que le daba dos opciones: 
+- [Morir luchando como sabia que pasaria en el futuro](time/time.md)
+- [Seguir sus instrucciones y ganar](time/time1.md)
```
Comment (casey-collab) [inline espanol/canudos.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: espanol/time/time.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Asi el consejero oprimio un boton de una extrana maquina que saco de su gaban y desaparecio mientras Beatito
```
Comment (casey-collab) [inline espanol/time/time.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: espanol/time/time1.md [hunk 3]
```diff
@@ -0,0 +1 @@
+El consejero apunto a un monte a lo lejos y dijo "lleva a tu ejercito a ese punto"...
```
Comment (casey-collab) [inline espanol/time/time1.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by tykiblood** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :fireworks:

**Pull request merged**